### PR TITLE
Add support for mobile Firefox old layout for google

### LINKED ISSUE #9: Trackers preview wheel covers the menu button on search result page
It looks that is some conditions google search result page renders the three-dot menu button, which is covered by our tracker wheel:

Zrzut ekranu 2022-02-7 o 10 03 35

### PR DESCRIPTION
**Pull request by smalluban**
Generally, I moved the wheel just after the URL, as sometimes there are additional features after, which can break the UI. Also, It fixes #9. After working with the result page for a few hours constantly, I think users might want to turn off the feature as it is too destructive. When I made a wheel slightly smaller and moved to the URL, now it looks more like a native google feature, rather than something strange. 

* Clean up of the UI code in content script
* Resized wheel to 16px - the 22px wheel was in my opinion too big, and hard to put in the line with the text (as we have limited options for styling it)
* One general styling without absolute positioning (JS calculated stuff)
* The script only checks the DOM where to put the wheel
* Fixed bug, when popup displays out of the screen because the wheel is to much to the left side of the window

Screenshots are from Chrome, but with different user agents.

Firefox Mobile
Zrzut ekranu 2022-03-3 o 12 02 37

Default Desktop (my personal account)
Zrzut ekranu 2022-03-3 o 12 03 08
Zrzut ekranu 2022-03-3 o 12 03 14

Default Mobile (my personal account)
Zrzut ekranu 2022-03-3 o 12 01 36

Default Mobile and desktop (another account)
Zrzut ekranu 2022-03-3 o 12 02 10
Zrzut ekranu 2022-03-3 o 12 02 20

### CODE REVIEW

File: packages/serp-report/src/content_scripts/serp-report.js [hunk 1]
```diff
@@ -54,76 +54,35 @@ function renderPopup(container, stats) {
...
-    container.style.top = 15 + 'px';
-  } else if (threeDotsElement) {
-    // default path on Safari (Desktop)
-    container.style.left =
-      threeDotsElement.getBoundingClientRect().right -
-      parent.getBoundingClientRect().left +
-      5 +
-      'px';
-  } else {
-    // default path in Chrome
```
Comment (philipp-classen) [inline packages/serp-report/src/content_scripts/serp-report.js:91]: Won't we break the functionality for Desktop in the ghostery-extension if we remove the logic for Chrome (Desktop)?

**Comment by chrmod** [comment]
by looking on the screenshots I think the wheel is little too high. The text number of trackers should share the baseline with the domain name.

**Review by chrmod — COMMENTED** [review]
Screenshot 2022-03-03 at 12 57 43

Does this PR cover Firefox for Android user agent? `Mozilla/5.0 (Android 12; Mobile; rv:68.0) Gecko/68.0 Firefox/97.0`

File: packages/serp-report/src/content_scripts/serp-report.js (hunk 1 above)
Comment (smalluban) [reply packages/serp-report/src/content_scripts/serp-report.js:91]: I've used the more appropriate place to put the wheel, so hacks with finding structure like above are no longer required.

**Comment by smalluban** [comment]
> by looking on the screenshots I think the wheel is a little too high. The text number of trackers should share the baseline with the domain name.

It is hard to make it perfect, as it is an inline-block with a different height than a text, but i can try to move it 1px less to the top. My main goal is to create more resistant styling (less CSS), so it will work fine even if they decide to change something there, so for example, no absolute positioning.

**Comment by smalluban** [comment]
> Does this PR cover Firefox for Android user agent? `Mozilla/5.0 (Android 12; Mobile; rv:68.0) Gecko/68.0 Firefox/97.0`

@chrmod - yes, the first screenshot is from Firefox Mobile user agent.

**Review by chrmod — APPROVED** [review]

**Review by philipp-classen — APPROVED** [review]

**Pull request merged**